specification/network/resource-manager/Microsoft.Network/stable/2023-11-01/networkWatcher.json

```diff
@@ -4273,6 +4274,14 @@
             "modelAsString": true
           },
           "description": "Test coverage for the endpoint."
+        },
+        "locationDetails": {
+          "$ref": "#/definitions/ConnectionMonitorEndpointLocationDetails",
```

Comment:
>ConnectionMonitorEndpointLocationDetails [](http://example.com/codeflow?start=33&length=40)

why will location details need more than just a region string?

Reply:
LocationDetails can also store availabilityZone. 

Connection Monitor provides unified, end-to-end connection monitoring in Azure Network Watcher. It monitors connectivity between two endpoints,. Endpoints can be of compound type like multiple machines in one endpoint. Incase of compound resource AzureArcNetwork, this location details would be needed